File: app/src/main/java/com/iamsmk/fynd/src/utils/FyndApplication.java
package com.iamsmk.fynd.src.utils;

import android.app.Application;


public class FyndApplication extends Application {


    private static FyndApplication _instance;

    @Override
    public void onCreate() {
        super.onCreate();
        _instance = this;
    }

    public static synchronized FyndApplication getInstance() {
        return _instance;
    }

}

File: app/src/main/java/com/iamsmk/fynd/src/utils/FyndApplication.kt
package com.iamsmk.fynd.src.utils

import android.app.Application
import com.google.android.material.color.DynamicColors

class MyApplication: Application() {
//    private val _instance: FyndApplication? = null
    override fun onCreate() {
        super.onCreate()
        DynamicColors.applyToActivitiesIfAvailable(this)
    }

//    @Synchronized
//    fun getInstance(): FyndApplication? {
//        return FyndApplication._instance
//    }
}